misc/selinux/cfengine-enterprise-unconfined.te

```diff
@@ -0,0 +1,8 @@
+module cfengine-enterprise-unconfined 1.0;
```

Comment:
Once this is functional I will add some comments here describing this file/module.